bundles/CoreBundle/Migrations/Version20200518205733

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
This file should have a `.php` extension.

Reply:
It was autogenerated from console via pimcore:migrations:generate but you're right